educator/tests/Test/Dscp/Educator/Web/Student/Queries.hs

```diff
@@ -157,27 +156,27 @@ spec_StudentApiQueries = describe "Basic database operations" $ do
                 _ <- createCourseSimple 2
                 CoreDB.enrollStudentToCourse student1 courseId1
 
-                course <- DB.getCourse student2 courseId1
-                return (not $ cIsEnrolled course)
+                course <- studentGetCourse student2 courseId1
+                return (not $ ciIsEnrolled course)
 
     describe "getCourses" $ do
         it "Student is not enrolled initially" $
             sqliteProperty $ \() -> do
                 _ <- CoreDB.createStudent student1
                 _ <- createCourseSimple 1
 
-                courses <- sqlTx $ DB.getCourses student1 Nothing
-                return (all (not . cIsEnrolled) courses)
+                courses <- sqlTx $ studentGetCourses student1 Nothing
+                return (all (not . ciIsEnrolled) courses)
```

Comment:
May be expressed as 
```
return $ not $ any ciIsEnrolled courses
```
To reduce amount of parenthes.

I don't insist here.

Reply:
Meh, right